lib/opte-test-utils/src/icmp.rs

```diff
@@ -0,0 +1,561 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+// Copyright 2023 Oxide Computer Company
```

Comment:
Most of this file is `oxide-vpc/tests/common/icmp.rs`, moved out to be usable from ubench.